packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -191,6 +191,22 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
     });
   };
 
+  private _elevateToMinimums(data: IChartDataPoint[]) {
+    let sumOfData = 0;
+    const minPercent = 0.01;
+    const elevatedData: IChartDataPoint[] = [];
+    data.forEach(item => {
+      sumOfData += item.data!;
+    });
+    data.forEach(item => {
+      elevatedData.push(
+        minPercent * sumOfData > item.data!
+          ? { ...item, data: minPercent * sumOfData, yAxisCalloutData: item.data!.toLocaleString() }
```

Comment:
the total sum could go beyond 100%. We need to renormalize this in the next pass.

Reply:
Since, the data gets modified prior passing to Pie.tsx, it gets normalized later on in Pie.tsx